pom.xml

```diff
@@ -44,7 +44,7 @@
         <swagger-annotations-version>1.6.5</swagger-annotations-version>
         <openapi-annotations-version>2.2.9</openapi-annotations-version>
         <!-- Used for liquibase maven plugin. Somehow use version from bom-internal instead -->
-        <postgresql.version>42.2.23</postgresql.version>
+        <postgresql.version>42.2.24</postgresql.version>
```

Comment:
Oh, I think what's going on based on the comment is that 
a) property scopes are different, the property in the bom probably only applies in the bom, this applies to other pom.xml, but so far that's only one place
b) removing  it, I get 
> [ERROR] 'build.plugins.plugin[org.liquibase:liquibase-maven-plugin].dependencies.dependency.version' for org.postgresql:postgresql:jar must be a valid version but is '${postgresql.version}'. @ io.dockstore:dockstore-webservice:${revision}${changelist}, /home/dyuen/dockstore/dockstore-webservice/pom.xml, line 1187, column 34

meaning, at some point we needed to override the postgresql dependency for the liquibase maven plugin only. Since that plugin is only used for proposing new migrations, there basically is no risk

Reply:
> meaning, at some point we needed to override the postgresql dependency for the liquibase maven plugin only. Since that plugin is only used for proposing new migrations, there basically is no risk

Oh, I see, it's a build plugin, so that must be why it doesn't show in the dependency tree in the test I did above.

Yeah, so it's an ephemeral PG instance that exists only during a build, so there is no risk. I feel it could just be done on develop, although it's harmless either way.